compiler/rustc_codegen_ssa/src/mir/mod.rs

```diff
@@ -199,6 +201,11 @@ pub fn codegen_mir<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
             }
         }
     }
+    if !all_consts_ok {
+        // There is no delay_span_bug here, we just have to rely on a hard error actually having been raised.
+        bx.abort();
```

Comment:
The test suite shows LLVM complaining "block ... does not have terminator". So looks like `abort` is not a terminator. Not sure what to put as a terminator though?